code/modules/particles/byond_particles/emitter/weather.dm

```diff
@@ -0,0 +1,27 @@
+/obj/emitter/rain
+	alpha = 255
+	particles = new/particles/rain
+	var/static/list/particles/z_particles
```

Comment:
Autodoc please

Reply:
dont think its necessary, its self explanatory its just not something that will occur in this codebase currently

Reply:
Autodoc is necessary for contributors reading this to know what this is supposed to be doing.